### DISCUSSION TITLE
"Automatically expire unpaid orders" does not happen, payment reminder email is not sent

### DISCUSSION THREAD

**Discussion by fabianhugo** [pretix/pretix#3560] [Q&A] [answered]
Hi dear all, 
first, thanks a lot for the work of this amazing software, I am a big fan. 
I use the latest version according to the small-scale docker deployment for 2 weeks, the event has a lot of placed orders. We only use the manual bank transfer payment option, and we noticed that the orders do not automatically expire after the deadline slips even though i checked the according box in Settings/Payment/Deadlines. 
Have i overlooked anything else? 
Also I notice that the automatic payment reminder emails do not get send, even though email delivery in all other cases works as expected. The number of days is set to 3 and we have text and subject for the case (if order will expire automatically). 
I checked docs.pretix.eu but can't get further. The server output did not show anything particularly strange.
Thanks a lot in advance,
Hugo

**Comment by pc-coholic** [comment, marked as answer]
Judging by the symptoms, I'd say that the cronjob is probably not set up properly: https://docs.pretix.eu/en/latest/admin/installation/docker_smallscale.html#cronjob

Would you mind verifying this?

**Reply by fabianhugo** [reply]
wow super fast reply, yes that was it, i forgot to add the cronjob. thanks a lot!
